docs/docs/projects/README.md

```diff
@@ -77,8 +79,72 @@ At this time it's not possible to move workflows out of GitHub Synchronized Proj
 
 :::
 
+## Access permissions
+
+### Understanding who has access to which project
+We've introduced 2 new columns in the [project list view](https://pipedream.com/projects): **Owner** and **Access**.
```

Comment:
```suggestion
The [project list view](https://pipedream.com/projects) contains **Owner** and **Access** columns.
```

Although it is a new feature, consider describing the feature as if it's been around, because eventually it will be and the docs won't change.